src/index.html

```diff
@@ -7,8 +7,32 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+        >
+      </div>
+      <nav class="nav">
+        <div class="menu">
+          <ul class="nav__list">
+            <li><a href="#" class="is-active">apple</a></li>
```

Comment:
```suggestion
            <li>
              <a href="#" class="is-active">apple</a>
            </li>
```
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md